packages/ra-ui-materialui/src/layout/Title.tsx

```diff
@@ -31,25 +32,25 @@ export const Title = (props: TitleProps) => {
 
     warning(!defaultTitle && !title, 'Missing title prop in <Title> element');
 
-    return (
-        <>
-            {createPortal(
-                <PageTitleConfigurable
-                    title={title}
-                    defaultTitle={defaultTitle}
-                    preferenceKey={preferenceKey}
-                    {...rest}
-                />,
-                container
-            )}
-        </>
-    );
+    const pageTitle =
+        preferenceKey === false ? (
+            <PageTitle title={title} defaultTitle={defaultTitle} {...rest} />
+        ) : (
+            <PageTitleConfigurable
+                title={title}
+                defaultTitle={defaultTitle}
+                preferenceKey={preferenceKey}
+                {...rest}
+            />
+        );
+
+    return <>{createPortal(pageTitle, container)}</>;
 };
 
 export interface TitleProps {
     className?: string;
     defaultTitle?: TitleComponent;
     record?: Partial<RaRecord>;
     title?: string | ReactElement;
-    preferenceKey?: string;
+    preferenceKey?: string | false;
```

Comment:
It would be nice to update the [documentation](https://github.com/marmelab/react-admin/blob/next/docs/Title.md#preferencekey) to indicate that this feature is now possible.